browser-test/src/currency.test.ts

```diff
@@ -60,15 +60,15 @@ describe('currency applicant flow', () => {
       await selectApplicantLanguage(pageObject, 'English')
 
       await applicantQuestions.applyProgram(programName)
-      const error = await pageObject.$('.cf-currency-value-error')
-      expect(await error.isHidden()).toEqual(true)
+      const currencyError = '.cf-currency-value-error'
+      expect(await pageObject.isHidden(currencyError)).toEqual(true)
```

Comment:
That's a great catch!

Apparently isHidden will return true if the selector doesn't match anything (https://playwright.dev/docs/api/class-page#page-is-hidden). 

I refactored to assert that the selector returns no elements instead here.

Reply:
Disregard my previous comment. We always add the error div and hide it based on whether it contains any validation errors (https://github.com/seattle-uat/civiform/blob/137b5980449567daf2444b345ef7691254eda21f/server/app/views/components/FieldWithLabel.java#L348). As such, we'd still want to assert it's hidden here (unless we refactored FieldWithLabel).

I've added a comment to this effect.

Reply:
not blocking: Not for this PR, but for future work - what do you think about us refactoring FieldWithLabel, so that we don't create the element at all? It seems a bit weird if we create the element and never unhide it.